check_tweets.py

```diff
@@ -0,0 +1,54 @@
+import mwclient
```

Comment:
you can use `mwrogue` library which is designed to support the esports wikis, and then:

```
from mwrogue.esports_client import EsportsClient
from mwrogue.esports_client import AuthCredentials
```

the 2nd line will allow you to log in, so that you can report errors on the wiki instead of printing them